### ISSUE TITLE
[Menu] - [Make checkbox menu keyboard accessible]

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#7382]
**Is this a new component or an extension of an existing one?**
What is the existing component, if any?
Menu 

**Describe the feature**
A clear and concise description of the new feature. What is the expected behavior?
the checkbox menu is not accessible via keyboard.
